### PR TITLE
Update setup_idf_chip.md

### PR DESCRIPTION
**Pull request by tianrking**
PR Summary:
Update the version number from ESP-IDF v5.1.2 to v5.3 in the setup_idf_chip.md file.
PR Details:

In the "Get IDF v5.1.2" section, change the version number from v5.1.2 to v5.3.
The rest of the content remains unchanged.

Signing the CLA:

I have carefully read and understand the terms of the Contributor License Agreement (CLA).
I confirm that I have the rights to the submitted content and agree to assign these rights to the project owners.
I have signed the CLA on the relevant webpage or document.

### CODE REVIEW

**Review by andy31415 — DISMISSED** [review]
Please update the PR summary (even if trivial, this summary makes it into our change log). Also please sign the CLA.

`Make sure you delete these instructions (to prove you have read them).`

**Comment by tianrking** [comment]
PR Summary:
Update the version number from ESP-IDF v5.1.2 to v5.3 in the setup_idf_chip.md file.
PR Details:

In the "Get IDF v5.1.2" section, change the version number from v5.1.2 to v5.3.
The rest of the content remains unchanged.

Signing the CLA:

I have carefully read and understand the terms of the Contributor License Agreement (CLA).
I confirm that I have the rights to the submitted content and agree to assign these rights to the project owners.
I have signed the CLA on the relevant webpage or document.

**Comment by bzbarsky-apple** [comment]
@dhrishi @shubhamdp

**Review by shubhamdp — APPROVED** [review]

File: docs/guides/esp32/setup_idf_chip.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ step.
 -   [Linux](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32/get-started/linux-macos-setup.html#for-linux-users)
 -   [macOS](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32/get-started/linux-macos-setup.html#for-macos-users)
 
-### Get IDF v5.1.2
+### Get IDF v5.3
```
Comment (dhrishi) [inline docs/guides/esp32/setup_idf_chip.md:19]: Now that we are already here, can you please change it to `Get ESP-IDF v5.3`? Thanks!

Comment (tianrking) [reply docs/guides/esp32/setup_idf_chip.md]: done ☺

**Review by dhrishi — APPROVED** [review]

**Comment by dhrishi** [comment]
@tianrking Thank you!

**Review by tianrking — COMMENTED** [review]
fix

**Review by saurabhst — APPROVED** [review]

**Review by ksperling-apple — APPROVED** [review]

**Review by tianrking — COMMENTED** [review]
FIX

**Review by tianrking — COMMENTED** [review]
FIX

**Pull request merged**